src/map/skill.cpp

```diff
@@ -15759,12 +15759,33 @@ int32 skill_castend_map (map_session_data *sd, uint16 skill_id, const char *mapn
 
 static bool skill_dance_switch(struct skill_unit* unit, bool revert);
 
-/// transforms 'target' skill unit into dissonance (if conditions are met)
+/**
+ * Turn a dissonance-overlapped skill unit back to its original form
+ * @param unit: Unit to transform back
+ */
+void skill_dance_overlap_revert(skill_unit* unit) {
```

Comment:
Please make it a reference.

Reply:
Can do it here, but I think for the actual overlap check it needs to be a pointer (I tried to make it a reference before but it threw errors, so I reverted to pointer, need to check again).